.changeset/silent-scissors-suffer.md

```diff
@@ -0,0 +1,9 @@
+---
+'@primer/brand-primitives': minor
+'@primer/react-brand': minor
+---
+
+⚠️ This update contains a breaking change.
```

Comment:
Is the breaking change still valid following recent changes? If it is, could we be more specific about whether it's a visual one vs API? Guessing it's the former, in which case we should give the user a nudge about what to look for. 

Example you can reference from another recent change: https://github.com/primer/brand/pull/616/files#diff-bf4a5c10ff09f2964e747a5873b4c1b2124a813202d314f69effbc3de1faf3dfR7-R8

Reply:
I've updated the changeset, let me know what you think 👍